### PR TITLE
Adding starting files

### PR DESCRIPTION
**Pull request by anthonymr**
# Milestone I - Hello World Page

In this PR, I did the following things :
- Added a .gitignore file.
- Installed linters successfully.
- Added descriptive README file.
- Added index.html
- Added style.css.
- Added test.md and ignored by gitignore
- Added end-of-source-newline

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+	<h1>Hello Microverse!</h1>
```
Comment (Qoosim) [inline index.html:11]: - [ ] It is required to give a class name to your `h1` tag and use that class name in your css file to style the content of the `h1` tag. Kindly add a class name and use that class name inside your css file to style the content of the `h1`.

project-req

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.main-header {
+  color: #4853e4;
+}
```
Comment (Qoosim) [inline style.css:3]: - [ ] It is required to use the class name given to your `h1` tag inside your index.html file to style the content of that tag. Kindly use that class name selector to reference that tag.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @anthonymr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (anthonymr) [reply index.html:11]: Ups, I forget it. I'm nervous because this is my first project.

**Pull request closed without merging**